app/src/main/java/org/groundplatform/android/common/Constants.kt

```diff
@@ -67,7 +67,12 @@ object Constants {
   // Issue URL: https://github.com/google/ground-android/issues/1730
   const val DEFAULT_MOG_TILE_LOCATION = "/offline-imagery/default"
   private const val DEFAULT_MOG_MIN_ZOOM = 8
-  private const val DEFAULT_MOG_MAX_ZOOM = 14
+  const val DEFAULT_MOG_MAX_ZOOM = 14
+  const val DEFAULT_MOG_UPSCALE_MAX_ZOOM = 15
```

Comment:
Why do we have 2 constants for the same thing? Can we reuse the existing one and remove the newly created one?